docs/prebuilt.md

```diff
@@ -55,6 +55,12 @@ Table S1 Confusion matrix for the Alive/Dead model in Weinstein et al. 2023
 
 Citation: Weinstein, Ben G., et al. "Capturing long‐tailed individual tree diversity using an airborne imaging and a multi‐temporal hierarchical model." Remote Sensing in Ecology and Conservation 9.5 (2023): 656-670.
 
+## Livestock Detectors
+
+The goal of this project is to detect livestock using drone and UAV imagery. To ensure that the model generalizes well across different points of view and camera types, various datasets will be used. This approach aims to enhance the model's ability to perform accurately under diverse conditions and perspectives.
```

Comment:
Don't use future tense, once its up on the website it should be 'available', so it reads. 

# Livestock Detector

This model has a single label 'livestock' trained on drone imagery of cows, sheep and other large mammals in agricultural settings. The model was trained on data from [insert countries and other metadata about landscapes].